NOTICE.txt

```diff
@@ -10,30 +10,37 @@ This product includes software developed by Aurelius (http://thinkaurelius.com/)
  * Marko A. Rodriguez
  * Stephen Mallette
  * Pavel Yaskevich
+ * Others as listed in CONTRIBUTORS.txt
```

Comment:
@sjudeng -- could you please file an issue to clean up the `NOTICE.txt` file and assign it to me, or update it in your PR?

* it should no longer be (c) Aurelius, but (c) JanusGraph Authors (listed in AUTHORS.txt); DataStax (which now owns Aurelius and hence their copyrights), is already listed in AUTHORS.txt
* instead of listing just a few individuals (who already appear in CONTRIBUTORS.txt), it should just say that it was developed by folks listed in CONTRIBUTORS.txt since it is more inclusive